Configuration/Geometry/README.md

```diff
@@ -52,5 +53,6 @@ Several detector combinations have been generated:
 * D51 = T17+C9+M4+I10+O4+F2
 * D52 = T18+C9+M4+I10+O4+F2
 * D53 = T15+C9+M4+I12+O4+F2
+* D60 = T19+C9+M4+I10+O4+F2
```

Comment:
not sure who can comment here, maybe @ghugo83 ?

Reply:
Yes, as mentioned in the commit, this should be renamed before making official PR.
This was done to limit conflicts in the private branch + could not know, at development time, which geometry scenario would be available at PR time anyway.
Need to simply switch D60 -> D54, this should be very easy to do.

Reply:
@ghugo83, do you want me to do the substitution. I can track the files where D60 is present, but maybe you can give me the list of files where to change it, in case it's not so straightforward to change.

Reply:
@jmduarte As you want, I can do it quickly if that helps you.
Otherwise go ahead, just a grep on D60 will do the job, because this geometry scenario did not exist before.

Reply:
it's fine, I'll do it. I just wanted to be sure that there aren't other dependencies apart of the D60 -> D54 change.

Reply:
@duartej A priori no, but there is sth worth looking up as well (workflow numbering in  Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py )
But I just checked and should be fine! 
I can do a commit, should take 1 min, or yes go ahead :)

Reply:
@ghugo83 , good thanks, I'm doing it now